_search-plugins/sql/protocol.md

```diff
@@ -329,3 +329,74 @@ Amber|Duke|32
 Dale|Adams|33
 Hattie|Bond|36
 ```
+
```

Comment:
Could we combine the "endpoint" and "protocol" files into one file and call it something like SQL and PPL API? We should also adhere more to our API style guide in terms of listing all of the query parameters and request fields. The query parameters are "format" and "sanitize". The request fields are "query", "fetch_size" and "parameters".

Reply:
@Yury-Fridlyand - do you mind if I take over this PR so that I can restructure the API section and address all of my comments?

Reply:
@kolchfa-aws,
No, I don't. Thank you for cooperation!